code/modules/mob/living/simple_animal/bot/honkbot.dm

```diff
@@ -278,15 +278,14 @@
 			else
 				continue
 
-/mob/living/simple_animal/bot/honkbot/explode()
+/mob/living/simple_animal/bot/honkbot/explode()	//doesn't drop cardboard nor its assembly, since its a very frail material.
 	walk_to(src, 0)
 	visible_message("<span class='boldannounce'>[src] blows apart!</span>")
 	var/turf/Tsec = get_turf(src)
-	//doesn't drop cardboard nor its assembly, since its a very frail material.
+	new /obj/item/bikehorn(Tsec)
+	new /obj/item/assembly/prox_sensor(Tsec)
```

Comment:
I'm pretty sure these are supposed to be under the 50% check.

Reply:
You are right,. Wanted to make it more consistent with other bots drop since the 50% drop is mostly for the robotic arm not other components

Added that to the CL